tests/topotests/bgp_vrf_ipv6_link_local/r1/bgpd.conf

```diff
@@ -0,0 +1,6 @@
+router bgp 65001 vrf blue
+  neighbor 192.168.255.2 remote-as 65002
```

Comment:
Can you remove this neighbor at all? You don't test it.

Reply:
done